task/package-operator-package/0.1/package-operator-package.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.21.1"
+    tekton.dev/tags: package-operator
+    tekton.dev/platforms: linux/amd64
+  labels:
+    app.kubernetes.io/version: "0.1"
+    build.appstudio.redhat.com/build_type: package-operator-package
+  name: package-operator-package
+spec:
+  description: |-
+    Given a git repository, a reference (as in, tag or commit) and a path within the
+    repository this task will create a package-operator package.
+
+    The process of how a pko package is defined and packaged is documented
+    [here](https://package-operator.run/docs/guides/packaging-an-application/).
+    This task expects the package definition, will build it using `kubectl-package`
+    and push the created package to the given OCI registry destination.
+  params:
+    - name: PACKAGE_PATH
+      description: File path the package manifest is in.
+      type: string
+    - name: SBOM_SRC
+      description: Source that is being specified in the SBOM pushed alongside the package.
+      type: string
+    - name: DST_URL
+      description: URL where to push the generated pko package to.
+      type: string
+  volumes:
+    - name: trusted-ca
+      configMap:
+        items:
+          - key: ca-bundle.crt
+            path: ca-bundle.crt
+        name: trusted-ca
+  stepTemplate:
+    volumeMounts:
+      - name: trusted-ca
+        mountPath: /etc/pki/tls/certs/ca-custom-bundle.crt
+        subPath: ca-bundle.crt
+        readOnly: true
+  steps:
+  - name: build-pkg
+    image: quay.io/redhat-services-prod/mos-lpsre-tenant/package-operator-internal/kubectl-package-internal:f02995b241a7d1a8afbc60b9dec233b03179328c7d4718c6d7360758d0b13794@sha256:f02995b241a7d1a8afbc60b9dec233b03179328c7d4718c6d7360758d0b13794
+    script: |
+      #!/bin/bash
+      set -eu -o pipefail
+      kubectl-package build --tag "$(params.DST_URL)" --push "$(workspaces.package.path)/$(params.PACKAGE_PATH)"
+  - name: sbom
```

Comment:
Just for visibility.. The PR in Mobster already exists and code below should be replaced by Mobster https://github.com/konflux-ci/mobster/pull/217